project01/paulo/apple.cc

```diff
@@ -0,0 +1,22 @@
+#include "apple.h"
+
+#include "game_canvas.h"
+
+static const ci::Color AppleColor{ci::Color::hex(0xDA1205)};
+
+Apple::Apple(GameCanvas& canvas) :
+  mCanvas(canvas)
+{
+  mLocation = mCanvas.GetRandomUnusedPoint();
+  mCanvas.Use(mLocation);
+}
+
+Apple::~Apple()
+{
+  mCanvas.Release(mLocation);
```

Comment:
Ok now I see it is needed... This Apple class is a bit too complicated, due to Canvas.